website/static/js/contribAdder.js

```diff
@@ -96,10 +96,17 @@ AddContributorViewModel = oop.extend(Paginator, {
         self.totalPages = ko.observable(0);
         self.childrenToChange = ko.observableArray();
 
+        self.emailSearch = ko.pureComputed(function () {
+            var emailRegex = new RegExp('[a-zA-Z0-9]+@[a-zA-Z0-9]+\.[a-z]+');
```

Comment:
We would never use this for real email address validation, but it should be fine for the purpose.